poppy/provider/akamai/certificates.py

```diff
@@ -565,15 +565,65 @@ def delete_certificate(self, cert_obj):
                 )
 
                 if found is False:
```

Comment:
Is the explicit use of False here because there could be a "truthy" value for found that we need to account for (instead of using `if not found`)?

Reply:
All the requested changes have been added in, could you please review the changes.